actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -766,12 +766,15 @@ public void validateFrameworkCategoryValues(
     return frameworkMap.get(key);
   }
 
+  // TODO:  Validate userType with data from form api
   private void validateUserType(Request userRequest) {
     String userType = (String) userRequest.getRequest().get(JsonKey.USER_TYPE);
 
     if (userType != null
```

Comment:
Add validation for sub type, if it is not null

Reply:
Cannot add currently as we dont have complete subtypes. Once form configuration is done we will change the validation from cache config.

Reply:
ok we do this validation during usertype form caching